zrml/swaps/src/benchmarks.rs

```diff
@@ -123,7 +128,7 @@ fn bench_create_pool<T: Config>(
         scoring_rule,
         if scoring_rule == ScoringRule::CPMM { Some(Zero::zero()) } else { None },
         if scoring_rule == ScoringRule::CPMM { Some(T::MinLiquidity::get()) } else { None },
-        if scoring_rule == ScoringRule::CPMM { Some(weights) } else { None },
+        if scoring_rule == ScoringRule::CPMM { some_weights } else { None },
     )
     .unwrap();
     let pool_id = <NextPoolId<T>>::get() - 1;
```

Comment:
`create_pool(...)` above returns the pool id, we should use that instead (to avoid storage operation).

Reply:
Mitigated in 1b280a1d152f2d2cc48622826130a8d9da1e846b